metagraph/src/annotation/representation/column_compressed/annotate_column_compressed.cpp

```diff
@@ -107,6 +144,44 @@ void ColumnCompressed<Label>::serialize(const std::string &filename) const {
         assert(column.get());
         column->serialize(outstream);
     }
+
+    if (!relation_counts_.size())
+        return;
+
+    outstream.close();
+
+    outstream.open(remove_suffix(filename, kExtension) + kExtension + ".counts",
+                   std::ios::binary);
+    if (!outstream.good())
+        throw std::ofstream::failure("Bad stream");
```

Comment:
I'd make this error message more useful by adding a `logger.trace("Could not open {}", filename);`
I sometimes get this "Bad stream" output in the integration tests, and it would be very useful if I knew which of the streams is actually bad.

Reply:
Ideally, we should go through all such error handlers and fix them all, maybe appending the filename to the exception message instead of writing to the logger.